src/server/helper/token.js

```diff
@@ -0,0 +1,19 @@
+import pkg from 'jsonwebtoken';
+const { jwt } = pkg;
+
+const header = {
+  "alg": "HS256",
+  "typ": "JWT"
+}
+
+export const generateToken = (payload) => jwt.sign(header, payload)
+
+export const generateExpiringToken = (payload, secret, expiryTime) => jwt.sign(payload, secret, { expiresIn: expiryTime })
+
+export const verifyToken = (webtoken, secret) => {
+  try {
+    return jwt.verify(webtoken, secret)
+  } catch (error) {
+    return false
```

Comment:
Better to actually let the error throw here (or throw your own custom error). Just returning false, you lose some of the error information. False tells us that the token wasn't valid, but it doesn't tell us _why_ the token wasn't valid. Is it malformed? is it expired? etc